app/assets/javascripts/modules/track-start-page-tabs.js

```diff
@@ -0,0 +1,23 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  'use strict'
+
+  function TrackStartPageTabs (module) {
+    this.module = module
+  }
+
+  TrackStartPageTabs.prototype.init = function () {
+    var tabs = this.module.querySelectorAll('.govuk-tabs__tab')
+
+    for (var i = 0; i < tabs.length; i++) {
+      tabs[i].addEventListener('click', function (event) {
+        var pagePath = event.target.href
+        GOVUK.analytics.trackEvent('startpages', 'tab', { label: pagePath, nonInteraction: true })
```

Comment:
I know this wasn't in the original file but normally we wrap calls to `trackEvent` in a check e.g.

```
if (GOVUK.analytics && GOVUK.analytics.trackEvent) {
    // call it
}
```
e.g. https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/javascripts/govuk_publishing_components/components/details.js#L28

I think this is probably for users who haven't consented to cookies - otherwise I think an error can occur.

Reply:
Oh yes, good point. Amended.